test/input_gen/genLayerTests.py

```diff
@@ -48,8 +48,13 @@ def inspect_file(file_name):
     record_single(fc, (3, 1, 1, 10), "fc_golden_plain")
     fc = K.layers.Dense(4)
     record_single(fc, (1, 1, 1, 10), "fc_golden_single_batch")
-
-# inspect_file("fc_golden.nnlayergolden")
-
-
+    bn = K.layers.BatchNormalization()
+    record_single(bn, (2, 4, 2, 3), "bn_golden_channels_training", {"training": True})
```

Comment:
consider increasing batch size more (just for batch normalization) - maybe 4?

Reply:
Will update

Reply:
sounds good. I will ensure to rebase #1590 after this merge and check with both batch size 2 and 4.